tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/annotations/AnnotationAttachmentPointTest.java

```diff
@@ -288,28 +288,9 @@ public void testInvalidAttachmentOnVar() {
         validateError(compileResult, index, "annotation 'v15' is not allowed on var", line + 3, 1);
     }
 
-    @Test
-    public void testInvalidAttachmentOnLetVar() {
-        int index = 152;
-        int line = 568;
-        validateError(compileResult, index++, "annotation 'v1' is not allowed on var", line, 13);
-        validateError(compileResult, index++, "annotation 'v2' is not allowed on var", line += 3, 13);
-        validateError(compileResult, index++, "annotation 'v3' is not allowed on var", line += 3, 13);
-        validateError(compileResult, index++, "annotation 'v4' is not allowed on var", line += 3, 13);
-        validateError(compileResult, index++, "annotation 'v5' is not allowed on var", line += 3, 13);
-        validateError(compileResult, index++, "annotation 'v6' is not allowed on var", line += 3, 13);
-        validateError(compileResult, index++, "annotation 'v7' is not allowed on var", line += 3, 13);
-        validateError(compileResult, index++, "annotation 'v8' is not allowed on var", ++line, 13);
-        validateError(compileResult, index++, "annotation 'v9' is not allowed on var", line += 3, 13);
-        validateError(compileResult, index++, "annotation 'v10' is not allowed on var", line += 3, 13);
-        validateError(compileResult, index++, "annotation 'v12' is not allowed on var", line += 3, 13);
-        validateError(compileResult, index++, "annotation 'v13' is not allowed on var", line += 3, 13);
-        validateError(compileResult, index, "annotation 'v15' is not allowed on var", line + 3, 13);
-    }
-
```

Comment:
Any reason why are these tests removed?

Reply:
yes, the syntax is correct. the test seems to be wrong.

Reply:
I think the test case is correct. We have to find and fix the root cause

Reply:
yep I ll get that fixed